### PR TITLE
Mv 437 parser

### PR DESCRIPTION
**Pull request by pkrucz00**
Parsing the form input and sending request to Google Form.

### CODE REVIEW

File: assets/src/features/leaving-page/components/Questionnaire.tsx [hunk 1]
```diff
@@ -5,14 +5,15 @@ import Rating from "./Rating";
...
+import sendForm from "../utils/sendForm";
 import { Controller, FieldError, SubmitHandler, UseFormRegister, useForm } from "react-hook-form";
 
 type RatingName = "video" | "audio" | "screenshare";
 
-type Inputs = {
-  video: number;
-  audio: number;
-  screenshare: number;
+export type Inputs = {
```
Comment (pkrucz00) [inline assets/src/features/leaving-page/components/Questionnaire.tsx:13]: Types and consts could be in one file for all questionnaire related files.

File: assets/src/features/leaving-page/utils/parseToQuery.tsx [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+type ResultType = {
+  [K in EntryName]: string;
+};
+
+const renameKeysToEntries = (formInput: Inputs): ResultType => {
+  const m = keyToEntryMap;
+
+  return Object.fromEntries(
+    Object.keys(formInput)
+      .filter((k) => k in formInput) //TODO merge this nad next line with type guard
```
Comment (pkrucz00) [inline assets/src/features/leaving-page/utils/parseToQuery.tsx:13]: Complete the TODO task

File: assets/src/features/leaving-page/components/Questionnaire.tsx [hunk 3]
```diff
@@ -5,11 +5,12 @@ import Rating from "./Rating";
 import Input from "../../shared/components/Input";
 import Button from "../../shared/components/Button";
 import Send from "../icons/Send";
+import sendForm from "../utils/sendForm";
 import { Controller, FieldError, SubmitHandler, UseFormRegister, useForm } from "react-hook-form";
 
 type RatingName = "video" | "audio" | "screenshare";
 
-type Inputs = {
+export type Inputs = {
```
Comment (ostatni5) [inline assets/src/features/leaving-page/components/Questionnaire.tsx:13]: Move export below import of icon

File: assets/src/features/leaving-page/components/Questionnaire.tsx (hunk 1 above)
Comment (pkrucz00) [reply assets/src/features/leaving-page/components/Questionnaire.tsx:13]: After refactor there is only 2 of them.

**Pull request merged**